draft-ietf-tls-tls13.md

```diff
@@ -2514,9 +2520,9 @@ certificate_request_context
   which will be echoed in the client's Certificate message. The
   certificate_request_context MUST be unique within the scope
   of this connection (thus preventing replay of client
-  CertificateVerify messages).   This field SHALL be zero length 
-  unless used for the post-handshake authentication exchanges 
-  described in Section 4.5.2. 
+  CertificateVerify messages).   This field SHALL be zero length
+  unless used for the post-handshake authentication exchanges
+  described in Section 4.5.2.
```

Comment:
Why is this referring to a section by number? Fix this with a section reference.